### PR TITLE
Added bugs for RN 4.10

### PR DESCRIPTION
**Pull request by ctauchen**
Bugs!

Applies only to `enterprise-4.10`

Preview: https://deploy-preview-42513--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -1204,6 +1204,26 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
...
+* Previously, the CRI-O runtime engine passed pod UIDs by using the `K8S_POD_UID` variable.
+But when pods were deleted and recreated at the same time that Multus was setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
+In this update, Multus handles pod UIDs, and unnecessary metadata processing is avoided.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2017882[*BZ#2017882*])
+
+* Previously, in deployments of OpenShift on a single node, machine config pools were not able to be updated because the SR-IOV Network Operator was unable to pause the machine config pool for the node it was assigned to.
+This fix allows the Operator to pause the machine config pool before running any configuration that requires a restart.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021151[*BZ#2021151*])
+
+* Client-go 1.20 and earlier did not have sufficient technique for retrying requests to the Kubernetes API.
```
Comment (skrthomas) [inline release_notes/ocp-4-10-release-notes.adoc:1222]: I'm wondering if this should be Client-go versions 1.20 and earlier..wdyt?

Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc]: Yes, that's much clearer. Fixed!

**Review by ctauchen — COMMENTED** [review]
@skrthomas! I made the change you suggested. I also rewrite the text for the bug on line 1218 (https://bugzilla.redhat.com/show_bug.cgi?id=2021151).

Can you look this over once more? Thanks!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1279,6 +1279,29 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
...
 * Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])
 
+* Previously, a webhook for the SR-IOV Network Operator blocked network policies for OpenShift installations on OpenStack.
+Users were not able to create SR-IOV network policies.
+This update fixes the webhook.
+Users can now create SR-IOV network policies for installations on OpenStack.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2016334[*BZ#2016334*])
+
+* Previously, the CRI-O runtime engine passed pod UIDs by using the `K8S_POD_UID` variable.
+But when pods were deleted and recreated at the same time that Multus was setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
```
Comment (skrthomas) [inline release_notes/ocp-4-10-release-notes.adoc:1289]:
```suggestion
But when pods were deleted and recreated in concurence with Multus setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
```

Comment (skrthomas) [reply release_notes/ocp-4-10-release-notes.adoc:1289]: this sentence bothered me last review, and I left it. However, this time, I'd like to suggest an alternative wording to shorten it a little.

**Review by skrthomas — COMMENTED** [review]
LGTM one suggestion, take it or leave it. I deliberated about it last time as well and decided to just put it.

**Pull request merged**